src/poisson.jl

```diff
@@ -110,17 +117,38 @@ dΓ = Measure(Γ,degree)
 #
 # With all the ingredients presented so far, we are ready to define the weak form. This is done by defining functions representing the bi-linear and linear forms:
 
+# From strong form to weak form (code viewpoint)
+#
+# In the strong form we have:
+#     -Δu = f in Ω,   u = g on Γ_D,   ∂u/∂n = h on Γ_N.
+#
+# To obtain the weak form, we multiply by a test function v (which vanishes on Γ_D) and integrate by parts, giving:
+#     ∫_Ω ∇u⋅∇v dΩ = ∫_Ω f v dΩ + ∫_{Γ_N} h v dΓ
+#
+# The code below encodes the left-hand side as a(u,v) and the right-hand side as b(v), using the same symbols ∇, dΩ, and dΓ as in the mathematical notation.
+
 f(x) = 1.0
 h(x) = 3.0
 a(u,v) = ∫( ∇(v)⋅∇(u) )*dΩ
 b(v) = ∫( v*f )*dΩ + ∫( v*h )*dΓ
+# Math ⇔ Code mapping:
+#   a(u,v) = ∫( ∇(v)⋅∇(u) )*dΩ      ↔    ∫_Ω ∇u⋅∇v dΩ
```

Comment:
Again, I do not see the point of this. The API is expressive enough that this "translation" is unnecessary, I think. If you look at both sides, the changes are minimal. I do not think people struggle with making this translation themselves.